docs/sources/setup-grafana/configure-security/configure-authentication/saml.md

```diff
@@ -421,6 +421,6 @@ Example of a configuration file:
 ...
 [security]
 csrf_trusted_origins = https://grafana.example.com
-csrf_origin_headers = X-Forwarded-Proto X-Forwarded-Host X-Forwarded-Port
+csrf_additional_headers = X-Forwarded-Host Referer
```

Comment:
Having both X-Forwarded-Host and Referer here doesn't really make sense. One is an alernative Host and the other an alternative Origin.

```suggestion
csrf_additional_headers = X-Forwarded-Host
```

It's confusing, because what I wanted to implement was probably aliases for Host, but what I ended up implementing was alternatives for Origin, which isn't something we necessarily need. Let's change that before this sees wide adoption, I think it's fair to call that oversight an implementation bug.

Reply:
I see. Thx for checking it so quick.